foundation/foundation-configuration-store/pom.xml

```diff
@@ -13,13 +13,25 @@
     <name>aiSSEMBLE::Foundation::Configuration::Store</name>
     <description>Provides centralized secure storage of configurations</description>
     <packaging>jar</packaging>
-
```

Comment:
S (Substantive): I don't think we want anything solution specific in foundation.  As such, I think we'd want to put this in something like 'extensions-configuration-store-vault`?

Reply:
Moved the store to vault specific logic to the extensions-configuration-store-vault